src/main/java/eu/europeana/metis/sandbox/service/dataset/DatasetReportServiceImpl.java

```diff
@@ -225,4 +226,8 @@ private void addStepInfo(List<ProgressByStepDto> stepsInfo,
     errorInfoDtoList.sort(Comparator.comparing(x -> x.getRecordIds().get(FIRST)));
     return errorInfoDtoList;
   }
+
+  private boolean isXsltNotEmpty(String xslt){
+    return StringUtils.isNotEmpty(xslt) && StringUtils.isNotBlank(xslt);
```

Comment:
You can just do isNotBlank. It checks for null, empty string and only spaces string.

Reply:
Good point, I will change this